aiidalab_widgets_base/utils/__init__.py

```diff
@@ -36,14 +36,14 @@ def predefine_settings(obj, **kwargs):
             raise AttributeError(f"'{obj}' object has no attribute '{key}'")
 
 
-def get_ase_from_file(fname, format=None):  # pylint: disable=redefined-builtin
+def get_ase_from_file(fname, file_format=None):  # pylint: disable=redefined-builtin
```

Comment:
This is a breaking change, are we fine with that? (I am using this in my app). Did you modify all the instance which call this function? I don't see it, but there should only be a few.

It is not a public method so perhaps it's fine.

Reply:
We are in the alpha-releases stage. So when if not now 😄